tavla/app/(admin)/boards/components/BoardTable/index.tsx

```diff
@@ -15,7 +19,11 @@ function BoardTable({
             <IllustratedInfo
                 title="Her var det tomt"
                 description="Du har ikke laget noen tavler ennå"
-                hasCreateButton={true}
+                CreateButton={() => (
```

Comment:
Jeg tror den riktige måten i React er å sende det som en `children`: https://react.dev/learn/passing-props-to-a-component#passing-jsx-as-children. @purusott har du noen peiling her?

Da kan `IllustratedInfo`-komponenten ta inn `children: React.ReactNode` som prop, også legger vi selve knappen under komponenten her, som dette:

```
<IllustratedInfo
                title="Her var det tomt"
                description="Du har ikke laget noen tavler ennå"
>
<PrimaryButton as={Link} href="?board">
                        Opprett tavle <AddIcon /> <CreateBoard />
                    </PrimaryButton>
</>
```

Hvis det ga mening?

Reply:
ja, stemmer, blir lettere å se hierarkien
har gjort noe liknende i min PR med expandable 👍

Reply:
skjønner! fiksa det nå tror jeg! btw, det funker å sette children sin type til både React.ReactNode og til JSX.Element. Ser at tidligere har vi brukt ReactNode så vi bør kanskje være consistent på det overalt, idk hvilken som er bedre

Reply:
kjør React.ReactNode ! JSX.Element er deprecated tror jeg uansett, og ReactNode gir større fleksibilitet for hva man kan sende inn som children

Reply:
nice, det er det jeg har brukt så da tror jeg denne er good